elife/postgresql-10.sls

```diff
@@ -0,0 +1,154 @@
+# postgresql-10.sls is intended to be a drop-in replacement for postgresql.sls (psql 9.4)
+# the two are mutually exclusive and share many of the same state names
+
+# copied from postgresql-client.sls
+
+{% set oscodename = salt['grains.get']('oscodename') %}
+
+# http://www.postgresql.org/download/linux/ubuntu/
+postgresql-deb:
+    pkgrepo.managed:
+        # http://www.postgresql.org/download/linux/ubuntu/
+        - humanname: Official Postgresql Ubuntu LTS
+        - name: deb http://apt.postgresql.org/pub/repos/apt/ {{ oscodename }}-pgdg main
+        - key_url: https://www.postgresql.org/media/keys/ACCC4CF8.asc
+
+# copied from 
+
+pgpass-file:
+    file.managed:
+        - name: /root/.pgpass
+        - source: salt://elife/config/root-pgpass
+        - template: jinja
+        - mode: 0600
+        - defaults:
+            user: {{ pillar.elife.db_root.username }}
+            pass: {{ pillar.elife.db_root.password }}
+            host: localhost
+            port: 5432
+
+{% if salt['elife.cfg']('cfn.outputs.RDSHost') %}
+pgpass-rds-entry:
```

Comment:
shouldn't this be the _only_ thing we do is RDS is present? We shouldn't need to install any package or setup any service as `postgresql-client.sls` should provide `psql`.

I may not be fully aware of use cases where RDS needs to run alongside a local `postgres` daemon though.

Reply:
we have one case that I've hated since forever: elife-dashboard (rds) + article-scheduler (local psql).

Reply:
I hate that one, too.